src/engraving/dom/dynamic.h

```diff
@@ -25,6 +25,8 @@
 
 #include "textbase.h"
 
+#include "draw/painter.h"
```

Comment:
Instead of including the header here, you could include it in the cpp file, and here in the header file only write 
```c++
namespace muse::draw {
class Painter;
}
```